### PR TITLE
Subtask/es 19/setup the env variables

### PR DESCRIPTION
**Pull request by coder-faraz**

### CODE REVIEW

File: sequelize.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+/**
+ * Specifying database configurations
+ * @param  {String} database name
+ * @param  {String} username
+ * @param  {String} password
+ * @param  {Object} cofiguration
+ * @return {Object} sequelize object
+ */
+const sequelize = new Sequelize('eden-sign', 'root', '', {
```
Comment (ahmadaafaq) [inline sequelize.js:11]: modify the creds from env

File: sequelize.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    host: "localhost",
+    port: 3306,
```
Comment (ahmadaafaq) [inline sequelize.js:13]: same here

**Review by ahmadaafaq — CHANGES_REQUESTED** [review]
Left comments

File: sequelize.js (hunk 1 above)
Comment (coder-faraz) [reply sequelize.js]: > please add the semicolons wherever missing

okay

**Comment by coder-faraz** [comment]
> Left comments

Will check and resend the PR

**Comment by coder-faraz** [comment]
> please add the semicolons wherever missing

okay

**Pull request merged**